book/src/concepts/world.md

```diff
@@ -115,6 +115,8 @@ All:
     world.delete_all().expect("Failed to delete all entities.");
 ```
 
+__Note: Entities are lazily deleted. They are effectively removed only at the end of each frame.__
```

Comment:
``Note: Entities are lazily deleted, which means that deletion only happens at the end of the frame and not immediately when calling the `delete` method.``
Or something like that.

Reply:
done